include/eosio/vm/execution_context.hpp

```diff
@@ -840,7 +838,7 @@ namespace eosio { namespace vm {
          (constants::max_call_depth + 1) * sizeof(activation_frame)
       };
       execution_state _state;
-      uint16_t                        _last_op_index    = 0;
+      uint32_t                        _last_op_index    = 0;
```

Comment:
Is there any consensus fallout for the various uint16_t->uint32_t changes in here?

Reply:
Actually yes: I didn't realize this originally, but it fixes an existing consensus bug.  `uint16_t` causes eos-vm interpreter to bail out in cases that jit (and OC) accept.  Since the behavior of jit is considered canonical, I believe that the interpreter should just be fixed to match it.  Unit test in test in the mandel repo incoming.